src/SASViyaApiClient.ts

```diff
@@ -610,7 +611,8 @@ export class SASViyaApiClient {
     parentFolderPath?: string,
     parentFolderUri?: string,
     accessToken?: string,
-    isForced?: boolean
+    isForced?: boolean,
+    serverUrl?: string
```

Comment:
`serverUrl` does not need to be passed in here - it is available as an instance variable in the `SASViyaApiClient` class.